chromadb/test/property/test_embeddings.py

```diff
@@ -91,7 +91,6 @@ def __init__(self, client: ClientAPI):
 
     @initialize(collection=collection_st)  # type: ignore
     def initialize(self, collection: strategies.Collection):
-        reset(self.client)
```

Comment:
this should be fine since the created collection is torn down after each state machine run
otherwise, the reset removes the non-default tenant and database that we created